config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "locals": {
+    "url": "http://engineering.underdog.io",
+    "name": "Underdog.io Engineering blog",
+    "owner": "Underdog.io Engineering",
+    "description": "Articles and insights from the Underdog.io Engineering team"
+  },
+  "plugins": [
+    "./plugins/paginator.coffee"
+  ],
+  "require": {
+    "moment": "moment",
+    "_": "underscore",
+    "typogr": "typogr"
+  },
+  "jade": {
+    "pretty": true
+  },
+  "markdown": {
+    "smartLists": true,
+    "smartypants": true
```

Comment:
Did some research. `wintersmith` uses `marked`. The version it's pointing to has `gfm` enabled by default:

https://github.com/jnordberg/wintersmith/blob/2.2.0/src/plugins/markdown.coffee#L4

https://github.com/jnordberg/wintersmith/blob/2.2.0/package.json#L35

https://github.com/chjj/marked/tree/v0.3.3#gfm